### ISSUE TITLE
Add Keyboard Accessibility

### ISSUE THREAD

**Issue by moarsel** [casesandberg/react-color#330]
By default, keyboard users are unable to use any of the color pickers that require clicking on a color (the input fields work fine). For example, the circle picker is unusable by keyboard because users cannot change focus or select a color, as far as I can tell. 

Two possible ways to go about fixing this include: 
1) Implement tabindex=0 on all relevant divs that need focus, and add a `Enter` key handler for selecting the colors. 
2) Implement the colors as a form element, such as radio buttons, that already have keyboard handling built in.

Both solutions would also require making some kind of keyboard focus style so the user can tell what is currently focused. If there is interest I might have time to try to fix this, but I just wanted to log the issue I'm having, since it breaks accessibility.

**Comment by casesandberg** [comment]
Just added tab index and focused styles in `react-color@2.12.1`
